cli/command/service/inspect_test.go

```diff
@@ -133,3 +145,14 @@ func TestJSONFormatWithNoUpdateConfig(t *testing.T) {
 	}
 	assert.Check(t, is.DeepEqual(m1, m2))
 }
+
+func TestPrettyPrintWithConfigsAndSecrets(t *testing.T) {
+	s := formatServiceInspect(t, formatter.NewServiceFormat("pretty"), time.Now())
+
+	if !strings.Contains(s, "Configs:") {
```

Comment:
You can use gotestyourself Contains assertion here:
```go
assert.Check(t, is.Contains(s, "Configs:"), "Pretty print missing configs")
assert.Check(t, is.Contains(s, "Secrets:"), "Pretty print missing secrets")
```

Reply:
Wondering if we should use the `.golden` files for this @vdemeester ?

Reply:
Yes, most likely we could use `golden` package for it :angel: 
Can be done in a follow-up though :wink: